content/docs/configuration.md

```diff
@@ -83,7 +83,30 @@ synced_files:
     dest: .
 ```
 
+## RPM spec file not in upstream repository
 
+If you do not want to have the RPM spec file in your upstream repository, for some reason,
+you can download it in [actions](/docs/actions/) section.
+
+Add [actions](/docs/actions/) section and
+hook `post_upstream_clone` into your `.packit.yaml` configuration file.
+Add row for downloading your RPM spec file from Fedora dist-git repository
+with your favorite command.
```

Comment:
Add a command for downloading your RPM spec file from Fedora dist-git repository.

Reply:
Fixed by commit https://github.com/packit-service/packit.dev/pull/36/commits/e877d65fe10018eca4d19bf6d2096d3a0bad3d72